### DISCUSSION TITLE
lastUpdated return from useData is not changed

### DISCUSSION THREAD

**Discussion by xu4wang** [vuejs/vitepress#1722] [Q&A] [answered]
I have two files, slides.md and home.md, they were modified on different date, but when calling useData(), it returns same time stamp.  Is this a bug or something wrong in my code? 

I am following https://github.com/vuejs/vitepress/issues/1450

The code I used:

```VUE
<script>
import { __pageData as foo } from './slides.md'

console.log(foo)

</script>
```

<img width="1081" alt="image" src="https://user-images.githubusercontent.com/311397/208869653-b028c006-afa9-46a2-927c-de3a8c2bd159.png">

**Comment by brc-dd** [comment, marked as answer]
You need to commit your code for `lastUpdated` to work. It uses git-based timestamp not fs-based.

**Reply by xu4wang** [reply]
thanks for the quick answer.

Is there a way I can get fs-based timestamp?

**Reply by brc-dd** [reply]
Probably will need to patch VitePress itself.

**Reply by xu4wang** [reply]
It will be a two or three lines of code patch.. I created a PR
https://github.com/vuejs/vitepress/issues/1723

not sure if it will be accepted.
